### PR TITLE
Add PolicyController sync SecurityPolicy from tower

### PR DESCRIPTION
**Pull request by zwtop**
* Add SecurityPolicy and IsolationPolicy types, and their Informers and Trackers.

* Add SecurityPolicy/Endpoint create/update/delete ClusterRole.

* Add PolicyController sync SecurityPolicy from tower.

* Add test cases for PolicyController.

### CODE REVIEW

File: plugin/tower/pkg/controller/policy/controller.go [hunk 1]
```diff
@@ -0,0 +1,820 @@
...
+			return nil, err
+		}
+		policy.Spec.IngressRules = ingress
+		policy.Spec.EgressRules = egress
+	}
+
+	return []v1alpha1.SecurityPolicy{policy}, nil
+}
+
+func (c *Controller) generateIntragroupPolicy(securityPolicyID string, appliedPeer *schema.SecurityPolicyApply) (*v1alpha1.SecurityPolicy, error) {
```
Comment (KeyongSun) [inline plugin/tower/pkg/controller/policy/controller.go:662]: If a "label" in the ApplyTo only contains 1 VM, do we need any specific process about the generate IntragroupPolicy?

Comment (zwtop) [reply plugin/tower/pkg/controller/policy/controller.go:668]: The vm relate labels are variable, we should just generate IntragroupPolicy, and don't care about the relation.

File: plugin/tower/pkg/controller/policy/controller.go [hunk 2]
```diff
@@ -0,0 +1,847 @@
...
+		}
+		if err == nil {
+			klog.Infof("delete policy %s", policyKey)
+		}
+	}
+
+	return nil
+}
+
+// parseIsolationPolicy convert schema.SecurityPolicy to []v1alpha1.SecurityPolicy
```
Comment (KeyongSun) [inline plugin/tower/pkg/controller/policy/controller.go:585]: parseSecurityPolicy

**Review by KeyongSun — APPROVED** [review]

File: plugin/tower/pkg/controller/policy/controller.go (hunk 2 above)
Comment (zwtop) [reply plugin/tower/pkg/controller/policy/controller.go]: done

**Pull request merged**